mir1350v5_1::
  #Controller to control robot arm joints
  abb_arm_controller:
    type: "position_controllers/JointTrajectoryController"
    joints: [abbLink2_revJoint, abbLink3_revJoint, abbLink4_revJoint, abbLink5_revJoint, abbLink6_revJoint,abbLink7_revJoint]

  #Controller to control end effector joints
  #eoat_controller:
  #  type: "position_controllers/JointTrajectoryController"
  #  joints: [joint8, joint9, joint10]

  #Controller to continuously publish joint states/positions
  joint_state_controller:
    type: "joint_state_controller/JointStateController"
    publish_rate: 50